Reminder-App/Reminder-App/ReminderDetailViewController.swift

```diff
@@ -9,62 +9,39 @@ import UIKit
 
 class ReminderDetailViewController: UITableViewController {
     
-    enum ReminderRow: Int, CaseIterable {
-        case title
-        case date
-        case time
-        case notes
-        
-        func displayText(for reminder: Reminder?) -> String? {
-            switch self {
-            case .title:
-                return reminder?.title
-            case .date:
-                return reminder?.dueDate.description
-            case .time:
-                return reminder?.dueDate.description
-            case .notes:
-                return reminder?.notes
-            }
-        }
-        
-        var cellImage: UIImage? {
-            switch self {
-            case .title:
-                return nil
-            case .date:
-                return UIImage(systemName: Images.calenderCirceImage)
-            case .time:
-                return UIImage(systemName: Images.clockImage)
-            case .notes:
-                return UIImage(systemName: Images.squareAndPencil)
-            }
-        }
-    }
-    
-    var reminder: Reminder?
+    // MARK: Private Propeeties
+    private var reminder: Reminder?
+    private var dataSource: UITableViewDataSource?
     
+    // MARK: Instance Methods
```

Comment:
See there is not - in MARK. Always be consistent with your coding style.
if you don't want - then remove from everywhere.